frontend/public/style.scss

```diff
@@ -10,6 +10,7 @@
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/variables";
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 @import "~font-awesome-sass/assets/stylesheets/font-awesome/variables";
+@import '~@patternfly/patternfly/sass-utilities/colors';
```

Comment:
I'm not sure this is right.

cc @rhamilto

Reply:
Why is this needed? We import `color-variables` above

Reply:
`color-variables` is pf3, `sass-utilities/colors` is pf4. Variable names are slightly different `$color-pf-black-100` (pf3) vs `$pf-color-black-100` (pf4)

Reply:
> Variable names are slightly different `$color-pf-black-100` (pf3) vs `$pf-color-black-100` (pf4)

I'm not sure what to recommend, but that's going to get really confusing :(

Reply:
I could use the pf3 ones but some variables have different values :/ But if want, i will use it, it shouldnt be a big change and once ready we can switch all css variables to pf4.

Reply:
No, I would use the PF4 variables here. Just thinking about how to deal with this. Maybe we should create `pf3-` aliases for legacy colors to avoid confusion, but that's outside the scope of this PR.

Reply:
Worth a comment in this file explaining, though.

Reply:
comment added